Remotion/Development/Web/UnitTesting/Configuration/WebConfigurationFactory.cs

```diff
@@ -15,31 +15,65 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
-using Remotion.Web.Configuration;
+using Remotion.Development.UnitTesting;
+using Remotion.ServiceLocation;
+using Remotion.Web.ExecutionEngine;
 
 namespace Remotion.Development.Web.UnitTesting.Configuration
 {
   public static class WebConfigurationFactory
   {
-    public static WebConfiguration GetExecutionEngineWithDefaultWxeHandler ()
+    public static ServiceLocatorScope GetBasicWxeUrlSettings ()
```

Comment:
Why do we have so many new methods in this class? They seem to be variants on the settings values so we should consider simplifying this API and removing the duplication.

We also need to update the names of the methods as they no longer "just get" something but rather open a new service locator scope.